documentation/IDTA-01001/modules/ROOT/pages/general.adoc

```diff
@@ -446,16 +446,23 @@ When comparing two elements, different use cases should be considered in order t
 This clause gives first hints on the aspects to consider when dealing with matching semantic identifiers.
```

Comment:
To someone that only reads this section (as they are only interested in how semantic identifiers are matched at that moment), it might seem like an official server would support all of these strategies. 
I see how this sentence, together with the fact that we're in the non-normative part of the specification should be enough to clarify that this isn't the case, but maybe it might make sense to add something like:

```suggestion
This clause gives first hints on the aspects to consider when dealing with matching semantic identifiers.
The actual implementations in applications may differ from the strategies presented here. 
```

Except if we actually think that "Exact Matching" is normative (considering we actually define equality here), but then it should be clearly stated so.